plugins/orchestrator-common/scripts/openapi.sh

```diff
@@ -8,9 +8,9 @@ DEFINITION_FILE="./src/auto-generated/api/definition.ts"
 METADATA_FILE="./src/auto-generated/.METADATA.sha1"
 
 openapi_generate() {
-    npx openapi-typescript ${OPENAPI_SPEC_FILE} -o ${SCHEMA_FILE}
-    npx openapi-generator-cli generate -g asciidoc -i ./src/openapi/openapi.yaml -o ./src/auto-generated/docs/index.adoc
-    npx yaml2json -f ${OPENAPI_SPEC_FILE}
+    npx --yes openapi-typescript@6.7.5 ${OPENAPI_SPEC_FILE} -o ${SCHEMA_FILE}
+    npx --yes @openapitools/openapi-generator-cli@v2.13.1 generate -g asciidoc -i ./src/openapi/openapi.yaml -o ./src/auto-generated/docs/index.adoc
+    npx --yes yaml2json -f ${OPENAPI_SPEC_FILE}
```

Comment:
What about a version for `yaml2json`?

Reply:
See my comment https://github.com/janus-idp/backstage-plugins/pull/1353#discussion_r1530457156